NEWS

```diff
@@ -1,3 +1,9 @@
+1.2.1.2
+  - set_github_pat() gains a `url` argument, which adds support for GitHub Enterprise.
+  - set_github_pat() now stores the PAT with actual username, instead of the "token" placeholder.
```

Comment:
This is a less obvious implication of what I've done here. It seems like a good idea to associate the actual username as a key with the credential, instead of the `"token"` placeholder. We're getting the username "for free" anyway. At least sometimes? See below.

Reply:
Here's a view into the macOS keychain before and after this PR. It feels more "correct" to me for the PAT to be associated with the correct username instead of just `"token"`. If we do continue to use a fake `username`, it feels like it should be a bit like a user agent and identify the R credentials package as the associated application.

<img width="296" alt="Screen Shot 2020-07-11 at 8 07 40 AM" src="https://user-images.githubusercontent.com/599454/87236932-5c5ef000-c3a4-11ea-958a-d451f62490dd.png">
<img width="307" alt="Screen Shot 2020-07-11 at 12 19 00 PM" src="https://user-images.githubusercontent.com/599454/87236935-64b72b00-c3a4-11ea-8cf3-ae84b775857b.png">

Reply:
Conclusion: the username **should** be a dummy placeholder to avoid interfering with any regular username/password that is stored.

But perhaps it should be more indicative that this token is being managed by the credentials R package?